pxr/usdImaging/usdImaging/lightAdapter.cpp

```diff
@@ -127,11 +163,14 @@ UsdImagingLightAdapter::TrackVariability(UsdPrim const& prim,
 
     UsdLuxLightAPI light(prim);
     if (TF_VERIFY(light)) {
-        UsdImaging_CollectionCache &collectionCache = _GetCollectionCache();
-        collectionCache.UpdateCollection(light.GetLightLinkCollectionAPI());
-        collectionCache.UpdateCollection(light.GetShadowLinkCollectionAPI());
-        // TODO: When collections change we need to invalidate affected
-        // prims with the DirtyCollections flag.
+        if (_UpdateCollectionsChanged(prim, cachePath))
+        {
+            *timeVaryingBits |= HdLight::DirtyBits::DirtyCollection;
+        }
+        else
```

Comment:
Can skip the else here.

Reply:
Actually, can just call _UpdateCollectionsChanged.  I don't think collections are allowed to be time-varying?  And if you did want to check that, instead of seeing if the collection has new contents, you'd want to ask whether the attributes are time varying.

Reply:
I removed the call altogether as the attributes cannot be animated... I'm guessing that if timesamples are added on the includeRoot bool that could cause some issues though?